tools/Hooks/pre-commit.sh

```diff
@@ -36,17 +36,14 @@ printf '%s\0' "${commit_files[@]}" | \
 # Use git-clang-format to check for any suspicious formatting of code.
 @CLANG_FORMAT_BIN@ --version > /dev/null
 if [ $? -eq 0 ]; then
-    clang_format_diffstat=$(@GIT_EXECUTABLE@ --no-pager \
-        clang-format --binary @CLANG_FORMAT_BIN@ --diffstat --quiet)
+    clang_format_diff=$(@GIT_EXECUTABLE@ --no-pager \
+        clang-format --binary @CLANG_FORMAT_BIN@ --diff --quiet)
```

Comment:
Even with `--quiet`, if there are no changes this returns

```
no modified files to format
```

Reply:
Ugh this works with clang-format v15

Reply:
Annoying that git-clang-format changed their behavior of `--quiet`